deploy/multiple_keys_migration.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy multiple_keys_migration
+--
+-- Populate keys table from users and clients table.
+BEGIN;
+--
+-- Initial migration
+-- The 'WHERE NOT EXISTS' clauses should prevent inserts when we already have a key.
+--
+
+LOCK TABLE keys IN SHARE ROW EXCLUSIVE MODE;
+  -- add clients
+  INSERT INTO keys
+    (id, key_id, public_key, key_version, created_at,  expires_at)
+	SELECT id, 'default', public_key, pubkey_version, updated_at, 'infinity'::timestamp FROM clients 
```

Comment:
Let's talk about pubkey_version and if we really need it. I'd like to move away from the branching on certificate(0?)/pub key (1?) and just force pub key.  Would this make sense as the right time to do that?